src/document.h

```diff
@@ -288,10 +289,12 @@ int Document_EvalExpression(RedisSearchCtx *sctx, RedisModuleString *key, const
 int Redis_LoadDocumentEx(RedisSearchCtx *ctx, RedisModuleString *key, const char **fields,
                          size_t nfields, Document *doc, RedisModuleKey **keyp);
 
+// Don't create document if it does not exist. Replace only
+#define REDIS_SAVEDOC_NOCREATE 0x01
```

Comment:
Isn't it the same as DOCUMENT_ADD_NOCREATE?

Reply:
`DOCUMENT_ADD_NOCREATE` means don't save the document at all. `REDIS_SAVEDOC_NOCREATE` means only replace the document if it already exists.